build.sbt

```diff
@@ -31,6 +31,23 @@ lazy val apso = (project in file("."))
   .dependsOn(akkaHttp, aws, caching, collections, core, elasticsearch, encryption, hashing, io, json, profiling, time)
   .aggregate(akkaHttp, aws, caching, collections, core, elasticsearch, encryption, hashing, io, json, profiling, testkit, time)
 
+
+lazy val docs = (project in file("apso-docs"))
+  .dependsOn(apso)
+  .settings(
```

Comment:
Shouldn't we also use `commonSettings` for the docs project?

Reply:
Yes, fixed on 510296efd350813041469c3d5306f1392635b906.